inappmessaging/USERGUIDE.md

```diff
@@ -397,6 +397,16 @@ override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<Str
 
 **<font color="red">Note:</font>** Please make sure that `trackPushPrimer()` API is only called for devices with Android 13 or higher OS since push notification permission request is not available for devices running in lower OS versions.
 
+### <a name="tooltip-campaigns"></a> #6 Tooltip Campaigns
```

Comment:
nit: add a note or link to this section in the SDK Integration section.

Reply:
Update